lib/ember-cli-fragments-import-paths.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const matchImport = require('./match-import');
+const EOL_REGEXP = require('./eol-regexp');
+const resolveCliAware = require('ember-cli-addon-aware-resolver');
```

Comment:
This should be supplied by the user.

In the eslint context they exist via configuration.